packages/guilded.js/lib/structures/User.ts

```diff
@@ -15,6 +19,8 @@ export class User extends Base<UserPayload> {
         this.name = data.name;
         this.createdAt = new Date(data.createdAt);
         this.type = data.type === "bot" ? UserType.Bot : UserType.User;
+        this.avatar = data.avatar;
```

Comment:
I’d move this prop and the below prop into a _update method that has some checks similar to here: https://github.com/guildedjs/guilded.js/blob/main/packages/guilded.js/lib/structures/Member.ts#L33